activities/ActivityTemplate/VueJS/js/components/SugarL10n.js

```diff
@@ -54,7 +62,15 @@ Vue.component('sugar-localization', {
 	methods: {
 		// Get a single string with parameters
 		get: function (str, params) {
```

Comment:
What happens when `params` is undefined/null?
Why not merge both `get`/`getString` methods in a simple `get` method?

Reply:
Sure, I think it'll work.